modules/shared/components/molecules/UploadingImage/UploadingImage.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import type { Story, Meta } from '@storybook/react';
+import UploadingImage from './UploadingImage';
+import type { IUploadingImage } from './IUploadingImage';
+
+export default {
+  title: 'molecules/UploadingImage',
+  component: UploadingImage,
+  argTypes: {},
+} as Meta;
+
+const Template: Story<IUploadingImage.IProps> = (args) => (
+  <UploadingImage {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
```

Comment:
why don't you pass the threedotsclickhandler here?

Reply:
we have never used a function in storybook controls, so it is not clear to me why I should do it.